src/main/java/org/junit/Assert.java

```diff
@@ -812,6 +812,37 @@ public static void assertNotSame(Object unexpected, Object actual) {
         assertNotSame(null, unexpected, actual);
     }
 
+    /**
+     * Asserts that two {@link CharSequence} instances have the same contentsa.
+     * If they do not, an {@link AssertionError} is thrown.
+     *
+     * @param expected the expected contents
+     * @param actual the object to compare to
+     */
+    public static void assertContentEquals(CharSequence expected, CharSequence actual)  {
```

Comment:
I think I would prefer a String. It would make it more likely that the person didn't get the parameters out of order, and it makes it more clear that, inline with assertEquals(), we won't fail if the CharSequences have different types.